lms/urls.py

```diff
@@ -970,12 +970,6 @@
         )
     ]
 
-# Branch.io Text Me The App
-if settings.BRANCH_IO_KEY:
```

Comment:
what else is BRANCH_IO_KEY used for?

Reply:
@sarina BRANCH_IO_KEY is being used [here](https://github.com/edx/edx-platform/blob/master/lms/templates/main.html#L165)

I tried to remove all new changes added in this [PR](https://github.com/edx-solutions/edx-platform/commit/efe4120b8fa67770ccd7a411dfd49ab03b1a7f1e)